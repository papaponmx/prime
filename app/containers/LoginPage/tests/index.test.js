import React from 'react';
import { shallow } from 'enzyme';
import { LoginPage } from '../index';

describe('<LoginPage />', () => {
  it('Should render component', () => {
    const wrapper = shallow(<LoginPage />);
    expect(wrapper).toMatchSnapshot();
  });

  it('Should render Google Button', () => {
    const wrapper = shallow(<LoginPage />);
    expect(JSON.stringify(wrapper.props().children, null, 2)).toMatch(
      'Sign in with Google',
    );
  });
});
