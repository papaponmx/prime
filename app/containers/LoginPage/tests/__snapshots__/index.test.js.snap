// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginPage /> Should render component 1`] = `
<div>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      LoginPage Prime Planner App
    </title>
    <meta
      content="Login into your potential"
      name="description"
    />
  </HelmetWrapper>
  <h1>
    <FormattedMessage
      defaultMessage="Welcome"
      id="app.containers.LoginPage.header"
      values={Object {}}
    />
  </h1>
  <GoogleButton
    disabled={false}
    label="Sign in with Google"
    onClick={[Function]}
    type="dark"
  />
  <p>
    <FormattedMessage
      defaultMessage="Let's get you started"
      id="app.containers.LoginPage.signInMessage"
      values={Object {}}
    />
  </p>
</div>
`;
